src/base.jl

```
@@ -30,14 +32,23 @@ const Components = Dict{DataType, Dict{String, <:Component}}
 
     # Keyword arguments
 
-    * `runchecks`::Bool : run available checks on input fields
+    * `runchecks`::Bool : run available checks on input fields. If an error is found in a field, that component will not be added to the system.
```

This should say that it will throw InvalidRange.